src/ProjectService.Models.Dto/Requests/CreateProjectImageRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Requests
+{
+    public class CreateProjectImageRequest
```

Comment:
i guess you can write simpler: "CreateImageRequest"

Reply:
and it should be record

Reply:
done